packages/@unimodules/react-native-adapter/android/src/main/java/org/unimodules/adapters/react/ReactNativeHostWrapper.kt

```diff
@@ -0,0 +1,152 @@
+package org.unimodules.adapters.react
+
+import android.app.Application
+import androidx.collection.ArrayMap
+import com.facebook.infer.annotation.Assertions
+import com.facebook.react.ReactInstanceManager
+import com.facebook.react.ReactNativeHost
+import com.facebook.react.ReactPackage
+import com.facebook.react.bridge.JSIModule
+import com.facebook.react.bridge.JSIModulePackage
+import com.facebook.react.bridge.JSIModuleSpec
+import com.facebook.react.bridge.JavaScriptContextHolder
+import com.facebook.react.bridge.JavaScriptExecutorFactory
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReactMarker
+import com.facebook.react.bridge.ReactMarkerConstants
+import com.facebook.react.common.LifecycleState
+import com.facebook.react.devsupport.RedBoxHandler
+import com.facebook.react.uimanager.UIImplementationProvider
+import java.lang.reflect.Method
+
+class ReactNativeHostWrapper(application: Application, host: ReactNativeHost)
+  : ReactNativeHost(application) {
+  private val host = host
```

Comment:
```suggestion
class ReactNativeHostWrapper(
  application: Application,
  private val host: ReactNativeHost
) : ReactNativeHost(application) {
```
You can inline property in the constructor.